src/Fhir.Anonymizer.Core/AnonymizerEngine.cs

```diff
@@ -20,21 +17,31 @@ namespace Fhir.Anonymizer.Core
     public class AnonymizerEngine
     {
         private readonly FhirJsonParser _parser = new FhirJsonParser();
-        private readonly PocoStructureDefinitionSummaryProvider _provider = new PocoStructureDefinitionSummaryProvider();
         private readonly ILogger _logger = AnonymizerLogging.CreateLogger<AnonymizerEngine>();
         private readonly ResourceValidator _validator = new ResourceValidator();
         private readonly AnonymizerConfigurationManager _configurationManger;
         private readonly Dictionary<string, IAnonymizerProcessor> _processors;
+        private readonly AnonymizationFhirPathRule[] _rules;
+
+        public static void InitFhirPathExtensionSymbols()
```

Comment:
Could we use "Initialize" here to keep consistency (like "InitializeProcessors")？

Reply:
Sure, updated.